### PR TITLE
Remove Delcross from reference

### LINKED ISSUE #1704: Remove dependencies on libraries in $ANSYSEM_ROOT_PATH/Delcross or remove warning about this folder
### Description of the feature

If $ANSYSEM_ROOT_PATH/Delcross is not in $LD_LIBRARY_PATH on Linux, PyAEDT will issue a warning.  This folder contains, among other libraries, Qt 5 libraries that may prevent the use of later Qt 5 versions by downstream apps that depend on PyAEDT.  In testing with such an app I found that at least EDB-only usage of PyAEDT seems to work without the Delcross folder in $LD_LIBRARY_PATH.

### Steps for implementing the feature

_No response_

### Useful links and references

_No response_

### PR DESCRIPTION
**Pull request by maxcapodi78**
Fix #1704

### CODE REVIEW

File: pyaedt/generic/clr_module.py [hunk 1]
```diff
@@ -30,11 +30,11 @@
         json_file = os.path.abspath(os.path.join(pyaedt_path, "misc", "pyaedt.runtimeconfig.json"))
         load("coreclr", runtime_config=json_file, dotnet_root=os.environ["DOTNET_ROOT"])
         print("DotNet Core correctly loaded.")
-        if "Delcross" not in os.getenv("LD_LIBRARY_PATH", "") or "mono" not in os.getenv("LD_LIBRARY_PATH", ""):
+        if "mono" not in os.getenv("LD_LIBRARY_PATH", ""):
             warnings.warn("LD_LIBRARY_PATH needs to be setup to use pyaedt.")
             warnings.warn("export ANSYSEM_ROOT222=/path/to/AnsysEM/v222/Linux64")
```
Comment (isaacansys) [inline pyaedt/generic/clr_module.py:35]: Should update this line for 232 as well, and maybe consider displaying the original message if on <232?

**Review by isaacansys — COMMENTED** [review]
These changes look good but we should also update the Linux CI to not include this folder for 232 so that the tests run with what we are recommending for users.

File: pyaedt/generic/clr_module.py (hunk 1 above)
Comment (maxcapodi78) [reply pyaedt/generic/clr_module.py:35]: at this point I don't know on which AEDT version the code is gonna run. so Either I put the message for everyone or I remove for everyone

Comment (maxcapodi78) [reply pyaedt/generic/clr_module.py]: @isaacansys  I've modified the message considering the option of 23.2. If you have a better idea let me know.

**Review by isaacansys — APPROVED** [review]
LGTM

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Pull request merged**